services/docker-compose.yml

```diff
@@ -503,7 +503,7 @@ services:
     image: ${DOCKER_REGISTRY:-itisfoundation}/webserver:${DOCKER_IMAGE_TAG:-latest}
     init: true
     hostname: "{{.Node.Hostname}}-{{.Task.Slot}}"
-    environment:
+    environment: &webserver_environment
```

Comment:
I have two observations here:
1. a lot of effort was put into isolating env vars for each individual service, if you have a new image we removed all this inheriting and manually have each entry listed. This allows people to more easily check env vars in a configuration file.
2. If you want to turn off certain parts of the "webserver API" that is not used by the "api-server" you cannot do it at the moment. Further more I would have expected to see parts of this turned off. Like maybe web socket handlers and stuff like that. I would only keep what you actually use in place.

Reply:
1. this service **is** a `webserver`, i.e. it has to behave exactly as the `webserver` therefore the env-vars will never change much (or in the worst case a _very few_ of them can be overwritten)
2. True, there are some functionality that will not be used which could be overcome by switching off some plugins. Nonetheless, if we do not do it to does not matter that much. The only possible overhead would be having some task running in the background that we might not need. 
   1. Actually the `web-socket` **must not** be disabled in this case. You are right that there is not front-end connected but it is **necessary** to be connected to the web-socket system via rabbit to notify e.g. payments etc.